website/content/docs/api-clients/client-agent.mdx

```diff
@@ -510,6 +510,13 @@ $ boundary client-agent pause
 
 Follow the troubleshooting steps to understand why the Client Agent is not able to reach the controller.
 
+#### WARNING! Remote host indentification has changed! It is possible that someone is doing something nasty!
+
+This issue typically arises when a user resumes the client-agent after a pause and attempts to connect to a target. 
+The error message is a result of the client-agent detecting a change in the remote host's identity due to change in remote worker IP address.
+
+HashiCorp advises removing the existing server host key from the `~/.ssh/known_hosts` file.
```

Comment:
It's probably worth having an entry for this here, but I would change the explanation a bit. What do you think about

```suggestion
#### WARNING! Remote host indentification has changed! It is possible that someone is doing something nasty!

This issue arises when using SSH to connect to an SSH target through an alias after the first successful connection to that alias. The issue is due to the way Boundary workers generate a new host key on every new SSH connection. The warning can safely be ignored using the `StrictHostKeyChecking=no` command line option:

```shell-session
$ ssh -o StrictHostKeyChecking=no targetalias.boundary.dev
```

You can also remove the existing server host key from the `~/.ssh/known_hosts` file to avoid the error.